src/vcs/git.rs

```diff
@@ -788,6 +789,55 @@ impl<'a> Browser<'a> {
         self.repository.revision_branches(&commit.id())
     }
 
+    /// Get the [`Stats`] of the underlying [`Repository`].
+    ///
+    /// # Errors
+    ///
+    /// * [`error::Error::Git`]
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use radicle_surf::vcs::git::{Browser, Repository};
+    /// # use std::error::Error;
+    ///
+    /// # fn main() -> Result<(), Box<dyn Error>> {
+    /// let repo = Repository::new("./data/git-platinum")?;
+    /// let mut browser = Browser::new(&repo)?;
+    ///
+    /// let stats = browser.get_stats()?;
+    ///
+    /// // there should be 5 branches (local & remote)
+    /// assert_eq!(stats.branch_count, 5);
+    ///
+    /// // there should be 14 commits
+    /// assert_eq!(stats.commit_count, 14);
+    ///
+    /// // 4 contributors: Alexander Simmerl, Fintan Halpenny, FintanH, Rudolfs Ošiņš
+    /// assert_eq!(stats.contributor_count, 4);
+    ///
+    /// # Ok(())
+    /// # }
```

Comment:
I think this kind of test makes more sense in a submodule as a unit test, rather than as a doctest, since it makes assumptions about the data available (`git-platinum`) and so might not make sense as documentation.

Reply:
All the doc tests/unit tests make assumptions about `git-platinum` data, though. The repository is treated as a set of "golden files".

Reply:
Ah indeed!